ansible/inventory/group_vars/operator-pipeline-integration-tests.yml

```diff
@@ -20,6 +20,10 @@ community_signing_pipeline_url: "{{ community_signing_pipeline_name }}-{{ oc_nam
 
 nginx_proxy_url: "{{ community_signing_pipeline_name }}-nginx-{{ oc_namespace }}.apps.pipelines-stage.0ce8.p1.openshiftapps.com"
 
+log_dir: /var/log/nginx
```

Comment:
Do we need to copy this to multiple inventories, if there is no change from operator-pipeline.yaml?

Reply:
Corrected.